src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -333,6 +282,46 @@ Public MustInherit Class BasicTestBase
             useLatestFramework:=useLatestFramework)
     End Function
 
+    Friend Shadows Function CompileAndVerifyEx(
```

Comment:
>Shadows [](start = 11, length = 7)

Is this shadowing another method?

Reply:
Hmm that commit got into this PR accidentally. I will remove.